audit_log/management/commands/clear_audit_log_entries.py

```diff
@@ -0,0 +1,22 @@
+from click import BaseCommand
+from django.conf import settings
+from logging import getLogger
+
+from audit_log.tasks import clear_audit_log_entries
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = (
+        "Clear AuditLog which is already sent to Elasticsearch,"
+        "only clear if settings.CLEAR_AUDIT_LOG_ENTRIES is set to True (default: False)"
+    )
+
+    def execute(self):
```

Comment:
Hmm, is this correct, shouldn't it be `handle()`?

Reply:
Yes, it should be handle, nice catch